lib/pushmi_pullyu/logging.rb

```diff
@@ -35,6 +35,23 @@ def self.log_aip_activity(aip_directory, message)
     aip_logger.close
   end
 
+  def self.log_preservation_event(deposited_file)
+    @preservation_logger ||= Logger.new("#{PushmiPullyu.options[:logdir]}/preservation_events.log")
+
+    message = "#{deposited_file.name} was successfully deposited into Swift Storage! \n"\
+    "Here are the details of this preservation event: \n"\
+    "NOID: '#{deposited_file.name}' \n"\
+    "Timestamp of Completion: '#{deposited_file.last_modified}' \n"\
+    "AIP Checksum: '#{deposited_file.etag}' \n"\
+    "Metadata: #{deposited_file.metadata} \n"
```

Comment:
If this is for human consumption, maybe add a `\t` to kick the details over and visually nest each event to make it easier to tell them apart? Just a thought.